src/Tickets/Blocks/Tickets/app/editor/container/template.js

```diff
@@ -46,6 +49,35 @@ const TicketsContainer = ({
 		return null;
 	}
 
+	const renderHelperText = () => (
+		<div className="tickets-row-line">
+			{
+				// translators: %s is the post type name.
+				sprintf( _x( 'Create single tickets for this %s. ', 'event-tickets' ), postType ? postType : '' )
+			}
+			<a className="helper-link " href="https://evnt.is/manage-tickets" target="_blank" rel="noopener noreferrer">{ __( 'Learn more about ticket management', 'event-tickets' ) }</a>
+		</div>
+	);
+
+	const renderPromptInfo = () => (
+		<div>
+			<Notice
+				description={
+					<div>
+						{ __( 'Create and manage Series Passes from the ', 'event-tickets' ) }
```

Comment:
We already have the same HTML built in the backend, [here](https://github.com/the-events-calendar/event-tickets/blob/0fafc1cac39350a894f2b03e0cc4712ef42daedc/src/admin-views/flexible-tickets/series-pass-event-notice.php#L13).
I think, in this case, having two versions would be beneficial in the long run to avoid having to move too much information back and forth, but this notice, as per my previous comment, should not live here.

Reply:
@bordoni I know this 2 translations approach is something you have suggested. But I'm not clear why using `sprintf`, from the `@wordpress/i18n` library, is not correct.

```js
const message = sprintf( __( 'Create and manage %1$s from the %2$s Series admin.', label, link ) );
```

Reply:
The issue with this approach is that we will need to use `dangerouslySetInnerHTML` and @bordoni told me before to not use it